reframe/frontend/executors/policies.py

```diff
@@ -26,6 +26,18 @@ def _cleanup_all(tasks, *args, **kwargs):
     # Remove cleaned up tests
     tasks[:] = [t for t in tasks if t.ref_count]
 
+def time_profiling(duration):
```

Comment:
I prefer sth like the following:

```suggestion
def format_duration(task):
```